docs/source/checkpoint.rst

```diff
@@ -35,3 +35,6 @@ torch.utils.checkpoint
 .. autofunction:: checkpoint
 .. autofunction:: checkpoint_sequential
 .. autofunction:: set_checkpoint_debug_enabled
+.. autoclass:: CheckpointPolicy
+.. autoclass:: SelectiveCheckpointContext
+.. autofunction:: gen_selective_checkpoint_context_fn
```

Comment:
Bikeshedding: What do you think of the name `create_sac_context_fn`. I don't quite like the prefix `gen` - it's somewhat ambiguous to me.

Reply:
Agreed on `gen`. Tentatively updating to`create_selective_checkpoint_contexts`:
- `sac` -> `selective_checkpoint` to avoid acronyms (we don't use AC anywhere)
- `context_fn` -> `contexts` to better reflect that we're actually returning 2 contexts.